front/src/styles/scss/common/components/_manchetteWithSpaceTimeChart.scss

```
@@ -0,0 +1,26 @@
+.manchette-space-time-chart-wrapper{
+    border-radius: .625rem;
+    box-shadow: 0rem .25rem .5625rem rgba(0, 0, 0, 0.06), 0rem .0625rem .125rem rgba(0, 0, 0, 0.19);
+    background-color: rgba(255, 255, 255, 1);
+
+
+    .header {
+      height: 2.5rem;
```

We miss a border-radius for the header

```suggestion
      height: 2.5rem;
      border-radius: 0.625rem 0.625rem 0px 0px;
```